lib/debounce.js

```diff
@@ -1,16 +1,11 @@
-import requestAnimationFrame from './requestAnimationFrame'
+export default function debounce (func, time = 300) {
+  let timeoutId = null
 
-export default function debounce (fn) {
-  let waiting = false
-
-  return function (...args) {
-    if (!waiting) {
-      waiting = true
-
-      requestAnimationFrame(() => {
-        fn(...args)
-        waiting = false
-      })
+  return function () {
```

Comment:
What was the reasoning behind this? The purpose of the previous implementation was to only invoke the function whenever a paint was about to happen. With the new implementation, you're delaying execution an arbitrary amount of time (possibly indefinitely, if the function is called repeatedly).